From db7054b68f295038dfcf2c211be353c19288e489 Mon Sep 17 00:00:00 2001
From: sfan5 <sfan5@live.de>
Date: Wed, 7 Sep 2022 21:01:26 +0200
Subject: [PATCH 3/7] Makefile: get rid of svn-related tools

---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index 63736f7..97e1d3f 100644
--- a/Makefile
+++ b/Makefile
@@ -110,6 +110,9 @@ install: all
 	for manfile in $(MANS); do \
 		install -Dm644 $$manfile -t $(DESTDIR)$(MANDIR)/man$${manfile##*.}; \
 	done;
+	for x in archco communityco archrelease commitpkg ${COMMITPKG_LINKS} crossrepomove ${CROSSREPOMOVE_LINKS} rebuildpkgs; do \
+		rm -f $(DESTDIR)$(PREFIX)/bin/$$x; \
+	done;
 
 uninstall:
 	for f in $(notdir $(BINPROGS)); do rm -f $(DESTDIR)$(PREFIX)/bin/$$f; done
-- 
2.37.3

